src/core/security/client_auth_filter.c

```diff
@@ -199,8 +200,28 @@ static void auth_start_transport_op(grpc_call_element *elem,
   channel_data *chand = elem->channel_data;
   grpc_linked_mdelem *l;
   size_t i;
+  grpc_client_security_context* sec_ctx = NULL;
```

Comment:
I'm not sure I understand what happens to this local `sec_ctx`. Is it referenced anywhere in the call after that?